### ISSUE TITLE
Prescribed dose

### ISSUE THREAD

**Issue by markbangert** [e0404/matRad#198]
Somewhere within matRad we should record the prescribed dose, among others for QI calculation.

**Comment by wahln** [comment]
Yes, I think we already pushed for that in issue #22 ... the question is where to put it  ;-)

**Comment by markbangert** [comment]
Considering the conventional matRad workflow this makes most sense for DICOM imports!

**Comment by wahln** [comment]
What do you mean with that? should it be only available for DICOM plans?

**Comment by markbangert** [comment]
I mean in matRad we dont really have the concept of dose prescription which somehow would also include a plan finalization or at least a renormalization to a point, volume, dose-volume, iso-dose etc. - In DICOM, however, there is such thing established. That is why this makes maybe more sense there.

**Comment by diazhekan** [comment]
Hello All, I am trying to figure out if "prescribed dose" is available in the most recent matrad package. Will you please give some guidance? Thanks for your help. Kan

**Comment by wahln** [comment]
No, we still do not explicitly handle "prescribed dose". We do not do that because most of the stuff related to prescribed dose can be handled manually with a few lines of code (e.g. dose normalization). Could you give me a short explanation for which purpose you would be interested in recording the prescribed dose?
